pkgs/steam-jupiter-original/default.nix

```diff
@@ -0,0 +1,23 @@
+# pkgs/games/steam/steam.nix with the Steam Deck client fat bootstrap
+#
+# This allows the Steam Deck UI to start on a fresh installation
+# (i.e., have not launched Steam at all before).
+
+{ steam-original, fetchurl }:
+
+let
+  fatBootstrap = "steam_jupiter_stable_bootstrapped_20220802.1.tar.xz";
+
+  bundle = fetchurl {
+    url = "https://steamdeck-packages.steamos.cloud/archlinux-mirror/sources/jupiter-main/steam-jupiter-stable-1.0.0.74-2.19.src.tar.gz";
```

Comment:
Yeah, not entirely sure what the best way is. We could traverse the pacman [package db tarball](https://steamdeck-packages.steamos.cloud/archlinux-mirror/jupiter-main/os/x86_64/jupiter-main.db.tar.xz), or do some ugly scraping of the HTML index. The name of the fat bootstrap tarball is in the PKGBUILD, but we can probably just use a wildcard.

Reply:
Right, using a wildcard would be better and only match the `$date.$rev` bit to be more resilient against future addition of additional tarballs.

And yeah, for the main package file (the one we `fetchurl`) since it's just a package in the repos, there's not much need to automate *right now* for only one package. Especially since AFAIUI this is only a "seed" and then the actual install would be installed from it, right? So even if it becomes old, it doesn't matter?

Reply:
Yeah, the bootstrap being old doesn't really matter that much. The fat bootstrap contains a full copy of the Steam client and Steam will try to self-update after selecting the Wi-Fi during the setup wizard.